packages/boot/test/fast-usdc/fast-usdc.test.ts

```diff
@@ -14,12 +14,33 @@ import {
   makeWalletFactoryContext,
   type WalletFactoryTestContext,
 } from '../bootstrapTests/walletFactory.js';
+import {
+  makeSwingsetHarness,
+  insistManagerType,
+} from '../../tools/supports.js';
+
+const test: TestFn<
+  WalletFactoryTestContext & {
+    harness?: ReturnType<typeof makeSwingsetHarness>;
+  }
+> = anyTest;
 
-const test: TestFn<WalletFactoryTestContext> = anyTest;
+const {
+  SLOGFILE: slogFile,
+  SWINGSET_WORKER_TYPE: defaultManagerType = 'local',
```

Comment:
I was expecting this to get set to `xsnap` somewhere in ci, but I don't see it.

I'm not certain where the bar should be. hm.